backend/app/conversations/callConvo.js

```diff
@@ -217,6 +217,139 @@ function howDidItGoConvo(user, message) {
   )
 }
 
+function noNextRepResponse(user, message, numCalls) {
+  return botReply(user, {
+    attachment: {
+      type: 'image',
+      payload: {
+        url: 'https://storage.googleapis.com/callparty/success.gif'
+      }
+    }
+  })
+  .then(() => {
+    // if the user is the first caller
+    if (numCalls <= 1) {
+      return botReply(user, stripIndent`
+        Congrats, you’re the first caller on this issue! You’ve joined the ranks of other famous firsts in American History. We'll reach out when we have updates and an outcome on the issue.
+      `)
+    }
+    // if the user is only person who has made calls, then it's weird to tell them how many calls so far so remove that part
+    else if (numCalls <= user.convoData.currentRepresentativeIndex + 1) {
```

Comment:
I'm not 100% sure that this condition will always mean that the user is the only person who has made calls. I agree that it's weird to tell the user how many calls have happened if they're the only person who has called, but this condition could evaluate to true in other situations, e.g. if the user has made one call, they have 3 calls remaining, and some other user has also made a call.

Maybe we could use `UserAction.count` to get the number of calls that the current user has made for the `CampaignCall`, and compare that to the total number of calls made for the `CampaignCall`? maybe that's overkill idk